src/primo/browzine-primo-adapter.js

```diff
@@ -278,6 +279,18 @@ browzine.primo = (function() {
     return articleRetractionUrl;
   };
 
+  function getArticleEOCNoticeUrl(scope, data) {
+    var articleEocNoticeUrl = null;
+
+    if (isArticle(scope)) {
+      if (data && data.expressionOfConcernNoticeUrl) {
+        console.log(data.expressionOfConcernNoticeUrl, 'here is our expressionOfConcernNoticeUrl');
```

Comment:
We can delete 🔥  this console log now that we're getting ready to merge this. Probably very helpful during debugging though 😊 

```suggestion
```